content/includes/L2-section-landing.html

```diff
@@ -1,26 +0,0 @@
-<div class="main" role="main">
-  <div class="section">
-    <div class="primary">
-      <div class="row">
-        <div class="small-12 columns usa-content">
-        {% if path == 'employment' %}
-          <div class="csp-inline-patch-L2-section-landing">
```

Comment:
😄 

Can you also remove the `csp-inline-path*` rules that were added to `src/sass/_shame.scss`?